sys-apps/ubus/ubus-9999.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils git-r3
+
+DESCRIPTION="An embedded bus system developed for OpenWrt. It is like dbus but simple and small."
+HOMEPAGE="https://git.openwrt.org/?p=project/ubus.git;a=summary"
+EGIT_REPO_URI="https://git.openwrt.org/project/ubus.git"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+IUSE="lua"
+
+DEPEND="
+	dev-libs/libubox
+	lua? ( dev-lang/lua )
+"
```

Comment:
Missing RDEPEND?

Reply:
Right, I had some changes in my overlay but forgot to sync those...
Thanks!